recipes/quickstart/Multi-Modal-RAG/README.md

```diff
@@ -0,0 +1,116 @@
+# End to End Tutorial on using Llama models for Multi-Modal RAG 
+
+## Recipe Overview: Multi-Modal RAG using `Llama-3.2-11B` model: 
+
+This is a complete workshop on labelling images using the new Llama 3.2-Vision Models and performing RAG using the image caption capiblites of the model.
+
+- **Data Labeling and Preparation:** We start by downloading 5000 images of clothing items and labeling them using `Llama-3.2-11B-Vision-Instruct` model
+- **Cleaning Labels:** With the labels based on the notebook above, we will then clean the dataset and prepare it for RAG
+- **Building Vector DB and RAG Pipeline:** With the final clean dataset, we can use descriptions and 11B model to generate recommendations
+
+## Requirements:
+
+Before we start:
+
+1. Please grab your HF CLI Token from [here](https://huggingface.co/settings/tokens)
+2. git clone [this dataset](https://huggingface.co/datasets/Sanyam/MM-Demo) inside the Multi-Modal-RAG folder: `git clone https://huggingface.co/datasets/Sanyam/MM-Demo`
+3. Launch jupyter notebook inside this folder
+4. We will also run two scripts after the notebooks
```

Comment:
this seems unnecessary to be part of the requirements?

Reply:
@HamidShojanazeri actually we need to clone the dataset from Huggingface, so im pointing that out. 

Do you think I should move it inside a notebook?